lib/isRetryableError.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
```

Comment:
can we rename this file, or ideally merge it with `error.js`? At very least it doesn't follow filename conventions in the module (we tend to use snake_case, all lowercase) It also just feels to specific to make it its own file - this is specific to errors, I would immediately think to look in the errors definition file

Reply:
updated to be part of `lib/error.js`